documentation/operators/src/nodes/mix-node-setup.md

```diff
@@ -59,64 +59,79 @@ To check available configuration options for initializing your node use:
 
 ~~~admonish example collapsible=true title="Console output"
 ```
- <!-- cmdrun ../../../../target/release/nym-mixnode init --help -->
+<!-- cmdrun ../../../../target/release/nym-mixnode init --help -->
 ```
 ~~~
 
-Initalise your mix node with the following command, replacing the value of `--id` with the moniker you wish to give your mix node. Your `--host` must be publicly routable on the internet in order to mix packets, and can be either an Ipv4 or IPv6 address. The `$(curl -4 https://ifconfig.me)` command returns your IP automatically using an external service. If you enter your IP address manually, enter it **without** any port information.
+Initialise your Mix Node with the following command, replacing the value of `--id` with the moniker you wish to give your Mix Node. Your `--host` must be publicly routable on the internet in order to mix packets, and can be either an Ipv4 or IPv6 address. The `$(curl -4 https://ifconfig.me)` command returns your IP automatically using an external service. If you enter your IP address manually, enter it **without** any port information.
```

Comment:
so we still have `--host` on the mix node, not `--listening-address`?

Reply:
yes, `--host` on mixnode and `--listening-address` on Gateway AFAIK

Reply:
done